acceptCard.py

```diff
@@ -74,3 +74,78 @@ async def acceptCard(
     if newCard:
         cardSheetUnapproved.update_cell(dbRowIndex, 1, cardName)
         cardSheetUnapproved.update_cell(dbRowIndex, 3, authorName)
+
+async def acceptVetoCard(
```

Comment:
maybe name it vetoAcceptedCard?

Reply:
This is the thing Liu asked for where the vetoed cards go into the graveyard so I figured it was like "accepting" a vetoed card